feature/amazon/src/main/java/com/revenuecat/purchases/amazon/ProxyAmazonBillingActivity.kt

```diff
@@ -0,0 +1,23 @@
+package com.revenuecat.purchases.amazon
+
+import android.app.Activity
+import android.os.Bundle
+import android.os.ResultReceiver
+import com.amazon.device.iap.PurchasingService
+
+class ProxyAmazonBillingActivity : Activity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        val sku = intent.getStringExtra("sku")
+        val resultReceiver = intent.getParcelableExtra("result_receiver") as? ResultReceiver
+
+        val requestId = PurchasingService.purchase(sku)
+
+        val bundle = Bundle().apply {
+            putParcelable("request_id", requestId)
+        }
+        resultReceiver?.send(0, bundle)
+
+        finish()
```

Comment:
Hmm we shouldn't need to do this I think? We will finish the activity from the `DefaultPurchasingServiceProvider` once `onPurchaseCompleted` is called. As in, I think right now we would be trying to finish this activity twice

Reply:
Oh yes, this is leftovers